backend/src/main/java/com/javahelp/backend/data/DynamoDBUserStore.java

```diff
@@ -36,14 +30,31 @@
  * - username: The string username of the user
  * - infotype: The type of the user, either CLIENT or PROVIDER
  * - salthash: Base64 string of the concatenated salt and hash (+ salt length)
- * - userinfo: lazy Base64 serialized UserInfo
+ * - user info attributes with corresponding names
  * <p></p>
  * The structure of the items in this table should not be depended on outside of this class
  * as elements, especially the Base64 encoded serialization of {@link UserInfo} may change in the
  * future
  */
 public class DynamoDBUserStore extends DynamoDBStore implements IUserStore {
 
+    private static final String CLIENT_UPDATE = "SET username=:unameval, " +
```

Comment:
I'm not really familiar with the use of `=:`, does `SET username=:unameval, ` mean `SET username` is defined as `unameval`?

Reply:
This is just the syntax for DynamoDB update item requests. `=:` is not an operator in this context (note that in some contexts `:=` is used as an assignment operator), rather the field named `username` is being assigned the value called `:unameval`.